source/db/nl-projects.json

```diff
@@ -4701,7 +4701,7 @@
     "description": "Veilige communicatiemiddelen voor activisten.",
     "license_url": "",
     "logo": "riseup.png",
-    "notes": "Riseup's services may also be accessed via their Tor Hidden Service addresses. A list is [available here](https://www.riseup.net/en/tor#riseups-tor-hidden-services).",
```

Comment:
Did you check if there are any instances of "hidden services"/"Hidden Services", plural?